### PR TITLE
[develop] Trello-gJnkflGa: Account creation. Value has initValue by default

### PR DESCRIPTION
**Pull request by AndreySurzhan**
[Trello-gJnkflGa](https://trello.com/c/gJnkflGa): Account creation. Value has initValue by default

### CODE REVIEW

File: src/controllers/account.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ module.exports = {
                 accounts: {
                     name: data.account.name,
                     initValue: data.account.initValue,
-                    value: data.account.value,
+                    value: _.isUndefined(data.account.value) ? data.account.initValue : data.account.value,
```
Comment (wildeastengineer) [inline src/controllers/account.js]: @AndreySurzhan, as far as I'm concerned,  we do not pass account `value` at all. Only`initValue`. But 'value' field of account will be fully calculated as a sum of `initValue` and all the transaction related to this account.

Comment (wildeastengineer) [reply src/controllers/account.js]: In this case it will fix [this ticket](https://trello.com/c/qaW5z5bS) too

Comment (AndreySurzhan) [reply src/controllers/account.js]: @wildeastengineer I get that 'value' should be calculated.
But so far we don't have transactions logic so during acc creation the 'value' has initValue value by default which is what you wanted, didn't you? 
It Also is by design that you cannot create acc without 'initValue'.

so could you please elaborate on your expectations please?

Comment (wildeastengineer) [reply src/controllers/account.js]: @AndreySurzhan as I understand now `value` should be just equal initValue.
`data` should not contain `value` field

**Pull request merged**